### PR TITLE
page-with-header-and-footer

### PR DESCRIPTION
**Pull request by priyanka9634**
page-with-header-and-footer

### CODE REVIEW

File: lib.ftd [hunk 1]
```diff
@@ -1238,33 +1234,30 @@ $sidebar-open: $header.sidebar-open
...
-
 -- ftd.row:
 width: fill-container
-padding-vertical.px: 31
-margin-left.px if { ftd.device == "mobile" }: 23
-margin-left.px if { ftd.device == "mobile" }: 23
+padding-vertical.px: 29
 spacing: space-between
-max-width.fixed.px: 1296
+max-width.fixed.px: 1320
```
Comment (gsalunke) [inline lib.ftd:1246]:
```suggestion
max-width.fixed.px: $max-width
```
add a global variable `max-width` and use it everywhere.

File: lib.ftd [hunk 2]
```diff
@@ -1238,33 +1234,30 @@ $sidebar-open: $header.sidebar-open
...
 
 -- ftd.row:
-width: fill-container
+align-self: center
+padding-vertical.px: 5
 
 -- ftd.image:
-if: {header-desktop.logo != NULL}
-src: $header-desktop.logo
+if: {header-desktop.site-logo != NULL}
```
Comment (gsalunke) [inline lib.ftd:1255]:
```suggestion
if: { header-desktop.site-logo != NULL }
```

File: lib.ftd [hunk 3]
```diff
@@ -1317,15 +1310,15 @@ border-color if {header-button.is-active}: $inherited.colors.cta-primary.hover
 border-radius.px: 50
 $on-mouse-enter$: $ftd.set-bool($a = $header-button.is-active, v = true)
 $on-mouse-leave$: $ftd.set-bool($a = $header-button.is-active, v = false)
-padding-vertical.px: 12
+padding-vertical.px: 11
 padding-vertical.px if {ftd.device == " mobile"}: 5
```
Comment (gsalunke) [inline lib.ftd:1314]:
```suggestion
padding-vertical.px if { ftd.device == " mobile" }: 5
```

File: lib.ftd [hunk 4]
```diff
@@ -1726,41 +1719,84 @@ align-self: center
...
 
 -- component footer:
-ftd.image-src logo: https://admint.io/assets/images/admint-logo-footer.svg
-footer-link list links: $footer-links
-social-icon list icons: $social-icons
-ftd.image-src bg-image: https://admint.io/footer-hand-thumb.48e6e2ea052bb6cf0beb.svg
+ftd.image-src site-logo:
+string site-url:
+pr.toc-item list footer-list:
+common.social-media list media:
```
Comment (gsalunke) [inline lib.ftd:1725]:
```suggestion
common.social-media list social-list: 
```

File: lib.ftd [hunk 5]
```diff
@@ -1726,41 +1719,84 @@ align-self: center
...
 
 -- ftd.column:
-colors: $cs.main
-types: $typo.types
 width: fill-container
 background.solid: $inherited.colors.cta-primary.base
-align-content: center
+
+-- footer-desktop:
+if: {ftd.device != "mobile"}
```
Comment (gsalunke) [inline lib.ftd:1733]:
```suggestion
if: { ftd.device != "mobile" }
```

File: lib.ftd [hunk 6]
```diff
@@ -1726,41 +1719,84 @@ align-self: center
...
+site-logo: $footer.site-logo
+site-url: $footer.site-url
+footer-list: $footer.footer-list
+media: $footer.media
+team-thumb: $footer.team-thumb
+
+-- footer-mobile:
+if: {ftd.device == "mobile"}
```
Comment (gsalunke) [inline lib.ftd:1741]:
```suggestion
if: { ftd.device == "mobile" }
```

File: lib.ftd [hunk 7]
```diff
@@ -1726,41 +1719,84 @@ align-self: center
...
+
+
+
+
+
+-- component footer-desktop:
+ftd.image-src site-logo:
+string site-url:
+pr.toc-item list footer-list:
+common.social-media list media:
```
Comment (gsalunke) [inline lib.ftd:1765]:
```suggestion
common.social-media list social-list: 
```

File: lib.ftd [hunk 8]
```diff
@@ -1726,41 +1719,84 @@ align-self: center
...
+string site-url:
+pr.toc-item list footer-list:
+common.social-media list media: 
+ftd.image-src team-thumb: 
 
 -- ftd.column:
-if: { ftd.device == "desktop" }
 width: fill-container
+align-self: center
 max-width.fixed.px: 1320
```
Comment (gsalunke) [inline lib.ftd:1771]:
```suggestion
max-width.fixed.px: $max-witdth
```

File: lib.ftd [hunk 9]
```diff
@@ -1769,48 +1805,53 @@ margin-top.px: 80
 align-self: end
 
 -- ftd.image:
-$loop$: $footer.icons as $obj
+$loop$: $footer-desktop.media as $obj
```
Comment (gsalunke) [inline lib.ftd:1808]:
```suggestion
$loop$: $footer-desktop.social-links as $obj
```

File: lib.ftd [hunk 10]
```diff
@@ -1769,48 +1805,53 @@ margin-top.px: 80
...
 
--- ftd.column:
-width: fill-container
-padding-left.px: 16
+
+-- component footer-mobile:
+ftd.image-src site-logo:
+string site-url:
+pr.toc-item list footer-list:
+common.social-media list media:
```
Comment (gsalunke) [inline lib.ftd:1834]:
```suggestion
common.social-media list social-links: 
```

**Review by gsalunke — CHANGES_REQUESTED** [review]
@priyanka9634 Plz change as per review comments.

File: common/fonts.ftd [hunk 11]
```diff
@@ -341,6 +341,28 @@ mobile: $montserrat-28-mobile
...
+-- ftd.type size-27-desktop:
+line-height.px: 27
+size.px: 18
+letter-spacing.px: 0
+weight: 600
+font-family: $nunito-assets.fonts.Nunito
+
+-- ftd.responsive-type size-27:
+desktop: $size-27-desktop
+mobile: $size-27-mobile
```
Comment (Arpita-Jaiswal) [inline common/fonts.ftd:360]:
```suggestion
-- ftd.type size-27-desktop:
line-height.px: 27
size.px: 18
letter-spacing.px: 0
weight: 600
font-family: $nunito-assets.fonts.Nunito

-- ftd.responsive-type size-27:
desktop: $size-27-desktop
```

Mobile type is not needed as it is similar to desktop

File: lib.ftd [hunk 12]
```diff
@@ -721,7 +718,9 @@ spacing.fixed.px: 24
 
 -- component slider:
 children wrap:
-integer $active-cell:
+integer $active-item:
+integer $max: 6
+integer $min: 1
```
Comment (Arpita-Jaiswal) [inline lib.ftd:723]: Why are these mutable?

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: lib.ftd [hunk 13]
```diff
@@ -548,49 +551,45 @@ color: $inherited.colors.cta-primary.text
...
-optional ftd.image-src logo: $assets.files.static.lezgoo.admint-logo.svg
-optional string contact-txt: Get Started🚧
-optional string contact-url: index.html
+string site-url: index.html
+ftd.image-src site-logo: $assets.files.static.lezgoo.admint-logo.svg
+string contact: Get Started🚧
+string contact-url: index.html
 pr.toc-item list sections: $sitemap.sections
 boolean $sidebar-open: false
+ftd.image-src footer-site-logo: $assets.files.static.footer.admint-logo-footer.svg
```
[the remark this answers is not in the document]
Comment (priyanka9634) [reply lib.ftd:560]: footer
header

both are different

**Pull request merged**